Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -90,6 +90,10 @@ namespace Equiv.Perm
 theorem mem_alternatingGroup {f : Perm α} : f ∈ alternatingGroup α ↔ sign f = 1 :=
   sign.mem_ker
 
+theorem swap_mul_swap_mem_alternatingGroup {g g' : Perm α} (hg : IsSwap g) (hg' : IsSwap g') :
```

Comment:
This should probably be `isSwap_mul_isSwap_mem_alternatingGroup` or `mul_mem_alternatingGroup_of_isSwap` since otherwise the lemma sounds like it's talking about `Equiv.swap`.